lisa/sut_orchestrator/azure/common.py

```diff
@@ -166,9 +166,8 @@ def get_storage_client(platform: Any) -> ComputeManagementClient:
     )
 
 
-def get_storage_account_name(platform: Any, location: str) -> str:
-    azure_platform: AzurePlatform = platform
-    subscription_id_postfix = azure_platform.subscription_id[-8:]
+def get_storage_account_name(subscription_id: str, location: str = "westus2") -> str:
```

Comment:
let's not put default value here. Let it be managed in higher level.

Reply:
Sure! I have updated it.

Reply:
@squirrelsc In the lsg storage notifier are we going to use "westus2" as the fixed location when calling this function or would `location` be a variable in yml file?

Reply:
@somil55, please do not mention any internal implementation here. :) For this case, it doesn't important, but others may be.
There shouldn't put a default location, because it depends on the deployed service. For our needs, it's westus2, but others may need eastus2. We can provide a default value in internal code or schema.

Reply:
@squirrelsc Sure! I will keep this in mind.